string home_folder
string current_folder
string selected_folder
string[] current_folders
string[] supported_file_types
int32 file_count

int32 max_files
string[] available_files_list
string[] selected_files_list

string[] current_file_list
string[] current_topic_list

bool pub_transforms
bool create_transforms

int32 max_pubs
float32[] min_max_delay
float32 set_delay

bool running






